mne/time_frequency/tfr.py

```diff
@@ -66,6 +66,10 @@ def morlet(sfreq, freqs, n_cycles=7.0, sigma=None, zero_mean=False):
     Ws = list()
     n_cycles = np.atleast_1d(n_cycles)
 
+    if np.any([f <= 0 for f in freqs]):
```

Comment:
I would mkae sure freqs is an array and do:

if np.any(freqs <= 0)

list comprehensions are slow and usually avoided in numericla code.

Reply:
@agramfort 
Okay, so:
A
```python
np.any( np.array(freqs) <= 0):
```
B
```python
freqs = np.array(freqs)
if np.any(freqs <= 0):
```
Or C?
Make sure users only can pass ndarrays (which is already true for ```tfr_morlet``` and ```tfr_multitaper``` but not for ```morlet```)

Reply:
I actually have a use case for only wanting to compute morlet (and multitaper I guess) at only one frequency. Better to cast to an array than fail in this case, I'd say.

Reply:
Done!